app/src/main/java/lt/vilnius/tvarkau/utils/FormatUtils.java

```diff
@@ -13,6 +14,12 @@
 
 public class FormatUtils {
 
+    private static final DateTimeFormatter EXIF_DATE_TIME = DateTimeFormatter.ofPattern("yyyy:MM:dd HH:mm:ss");
+
+    public static String formatExifAsLocalDateTime(@NonNull String exifDateTime) {
```

Comment:
Why always ? I see no reason to have try/catch in `NewReportFragment.kt` just because this function might throw exception.